src/app/shared/components/multi-select/multi-select.component.ts

```diff
@@ -0,0 +1,49 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core'
+import { IconButton, IconButtonSize } from '../../enums/icon-button'
+
+export interface MultiSelectOptions {
+  label: string
+  id: string
+  classesOnSelected?: string
+  classesOnList?: string
+}
+
+@Component({
+  selector: 'sofie-multi-select',
+  templateUrl: './multi-select.component.html',
+  styleUrls: ['./multi-select.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MultiSelectComponent {
+  @Input() public options: MultiSelectOptions[]
+  @Input() public selectedOptionsIds: string[]
+  @Input() public placeholder: string
+  @Input() public showOnlyItemsCountLabel?: string
+
+  @Output() public readonly onItemSelectionChange: EventEmitter<string[]> = new EventEmitter<string[]>()
+
+  public readonly IconButton = IconButton
+  public readonly IconButtonSize = IconButtonSize
+
+  public getOptionById(optionId: string): MultiSelectOptions | undefined {
+    return this.options.find(option => option.id === optionId)
+  }
+
+  public isOptionSelected(option: MultiSelectOptions): boolean {
+    return !!this.selectedOptionsIds.find(selectedOptionId => selectedOptionId === option.id)
+  }
+
+  public onOptionClick(option: MultiSelectOptions): void {
+    const selectedOptionIndex: number = this.selectedOptionsIds.findIndex(selectedOptionId => selectedOptionId === option.id)
+    if (selectedOptionIndex !== -1) {
+      this.unselectOption(selectedOptionIndex)
+    } else {
+      this.onItemSelectionChange.emit([...this.selectedOptionsIds, option.id])
+    }
+  }
+
+  public unselectOption(optionIndex: number): void {
```

Comment:
This is very nerdy and i'm probably the only one who cares, so no hard feelings if this is how we do it. 

"Unselect", i believe, generally applies to things that has yet to be selected, where "deselect" refers to removing actual selections.